recommendation/pytorch/README.md

```diff
@@ -112,7 +125,8 @@ The author's original code is available at [hexiangnan/neural_collaborative_filt
 Hit rate at 10 (HR@10) with 999 negative items.
 
 ### Quality target
-HR@10: 0.635
+HR@10: 0.635 (ml-20m)
```

Comment:
I would delete this line, as MLPerf will no longer use the small data set.

Reply:
Done.